config.json

```diff
@@ -23,6 +23,14 @@
     "rp10": 910319453491839069,
     "patch": 445773142233710594
   },
+  "grantableRoles": {
```

Comment:
Would it be feasible to just take these from where they're already defined for the text commands? https://github.com/Erisa/Cliptok/blob/8abca0cd60ce38c1888d5b218e8c7beb1f9b3d16/config.json#L225-L232

Reply:
Hmm I don't recall that ever existing back when I started writing that command or at least it was named in a way that says it's specifically for Insider roles, not for any other kind so that's why I added a grantable roles property.

Reply:
Afaik its been the same since 2020, just added more roles: https://github.com/Erisa/Cliptok/pull/2/files#diff-04f215c4509094fd47ad977ad739979ecc497a5801ec821ff3045476058a193aR30
But yeah, I believe it's fine to reuse this here. The old text user role commands will probably be phased out at some point anyway with this + discords onboarding feature

Reply:
Oh yeah I only saw `announcementRoles`, haven't noticed this one though.

Reply:
Should be done now.